docs/page/adding-new-language.md

```diff
@@ -19,7 +19,7 @@ corresponding mode -\> language id - in this case `(python-mode .
 
```

Comment:
can you drop somewhere a note, something like: 

`lsp-mode` is using `lsp-language-id-configuration` to determine what is the buffer language. When the major-mode is not sufficient to determine the language (e. g. web-mode is used for javascript and for html) you can put regex.

Reply:
Done